xclim/locales/fr.json

```diff
@@ -1,26 +1,95 @@
 {
   "attrs_mapping": {
-    "modifiers": ["m", "f", "mpl", "fpl"],
-    "YS": ["annuel", "annuelle", "annuels", "annuelles"],
-    "MS": ["mensuel", "mensuelle", "mensuels", "mensuelles"],
-    "QS-DEC": ["saisonnier", "saisonnière", "saisonniers", "saisonnières"],
-    "DJF": ["hivernal", "hivernale", "hivernaux", "hivernales"],
-    "MAM": ["printanier", "printanière", "printaniers", "printanières"],
-    "JJA": ["estival", "estivale", "estivaux", "estivales"],
-    "SON": ["automnal", "automnale", "automnaux", "automnales"],
-    "norm": ["normal", "normale", "normaux", "normales"],
-    "m1": ["janvier"],
-    "m2": ["février"],
-    "m3": ["mars"],
-    "m4": ["avril"],
-    "m5": ["mai"],
-    "m6": ["juin"],
-    "m7": ["juillet"],
-    "m8": ["août"],
-    "m9": ["septembre"],
-    "m10": ["octobre"],
-    "m11": ["novembre"],
-    "m12": ["decembre"]
+    "modifiers": [
```

Comment:
This is ugly. Is black overdoing it ?

Reply:
Black shouldn't be touching this as it's a json file. This is most likely due to an IDE reformatting operation.

Reply:
@aulemahal is currently fixing this. The fix should be pushed soon.

Reply:
This is my fault-ish. I use `xc.locales.generate_local_dict` and overwrite the previous file with `json.dump(****, indent=2)`. Which does that.

Reply:
I will open another PR for a nice solution to this.

Reply:
10-4